test/jdk/java/lang/module/ModuleDescriptorHashCodeTest.java

```diff
@@ -73,4 +171,23 @@ private static ModuleDescriptor fromModuleInfoClass(Module module) throws IOExce
             return ModuleDescriptor.read(moduleInfo);
         }
     }
+
+    // creates a module descriptor with passed (optional) opens/exports/requires modifiers
+    private static ModuleDescriptor createModuleDescriptor(
+            Set<ModuleDescriptor.Opens.Modifier> opensModifiers,
+            Set<ModuleDescriptor.Requires.Modifier> reqsModifiers,
+            Set<ModuleDescriptor.Exports.Modifier> expsModifiers) {
+
+        final var mdb = ModuleDescriptor.newModule("foobar");
```

Comment:
Does "mdb" stand for something here?

Reply:
Hello Alan, that was supposed to mean module descriptor builder. I will rename it to `builder` to be clear.